tests/integration/interactive_map_spec.js

```diff
@@ -29,7 +29,26 @@ describe("The web app", function() {
       .should("have.css", "display", "none");
   });
 
-  it("displays the timelapse components in their default states", function() {
+  it("updates map legend when another layer is selected", function() {
```

Comment:
Re-arranging the tests to make them pass is a bad idea; tests should be able to run independently from one another and still pass. To make the time-lapse components appear, you can issue a click event for the inundation layer at the beginning of your test.